packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -624,7 +615,6 @@ export abstract class Chart extends Observable implements AgChartInstance {
 
             case ChartUpdateType.PROCESS_DATA:
                 await this.processData();
-                this.disablePointer(true);
```

Comment:
NOTE: The use-case here is that the highlighted node + tooltip should update when the data changes underneath it - one of the real-time examples is a good test-case for this.

Reply:
Okay. Thanks for the info. I'll restore these. Although I think that renaming the function to something like `resetPointer()` might make more sense.